CMakeLists.txt

```diff
@@ -111,12 +111,18 @@ set(Slicer_EXTENSION_SOURCE_DIRS
   )
 
 # Add Eigen3 (required by at least SlicerSkeletalRepresentation)
+# Disable testing of Eigen
+option(EIGEN_BUILD_TESTING "Enable/Disable BUILD_TESTING in Eigen3" OFF)
+mark_as_advanced(EIGEN_BUILD_TESTING)
```

Comment:
instead of an option, I suggest to make it even more advanced with:
```
if(NOT DEFINED EIGEN_BUILD_TESTING)
  set(EIGEN_BUILD_TESTING OFF)
endif()
```